### PR TITLE
doc: Update doc for v3

### LINKED ISSUE #516: Update README
Add instructions for configuring app name using env variable. (Implemented in PR #501)

Taking this up.

### PR DESCRIPTION
**Pull request by mejariamol**
- added instruction to configure app name
- added google play store badge to README

fixes #516

### CODE REVIEW

**Comment by mejariamol** [comment]
@iamareebjamal please review.

File: README.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ The core features of this Android Application are
 - Ticket Management
 - Data sync with the [Open Event Organizer Server](https://github.com/fossasia/open-event-orga-server/)
 
+Currently, the application is released in alpha phase at Google Play Store [here](https://play.google.com/store/apps/details?id=org.fossasia.eventyay&ah=iHmAib1v0y1ER3l9rcx8YNEQAWU).
```
Comment (iamareebjamal) [inline README.md]: Remove the ah query tag

**Comment by mejariamol** [comment]
>Also change codecov badge to development branch

codacy badge should be changed to dev branch. and I think I don't have access for that. How is that done? @iamareebjamal

**Comment by iamareebjamal** [comment]
Leave codacy for now, nobody has access to that account. Change codecov one

**Comment by mejariamol** [comment]
@iamareebjamal please review.

**Pull request merged**